test.js

```diff
@@ -60,3 +60,158 @@ function myFunction2() {
+  },
+  {
+    name: 'The Hardest Project',
+    title: 'Multi-Post Stories Gain+Glory 4',
+    imgsrc: './images/Snapshoot.png',
+    imgMain: './images/Img_placeholder.png',
+    technologies: ['Node.js', 'Javascript', 'Bootstraps'],
+    techMain: ['css', 'Javascript', 'html'],
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  },
+  {
+    name: 'Milestone Project',
+    title: 'Multi-Post Stories Gain+Glory 5',
+    imgsrc: './images/Snapshoot.png',
+    imgMain: './images/Img_placeholder.png',
+    technologies: ['css', 'Javascript', 'html'],
+    techMain: ['css', 'Javascript', 'html'],
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  },
+  {
+    name: 'Game Project',
+    title: 'Multi-Post Stories Gain+Glory 6',
+    imgsrc: './images/Snapshoot.png',
+    imgMain: './images/Img_placeholder.png',
+    technologies: ['css', 'Javascript', 'html'],
+    techMain: ['css', 'Javascript', 'html'],
+    description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  },
+];
+
+const container = document.getElementById('whole-works-section');
+
+const works = document.createElement('div');
+works.setAttribute('class', 'desk-works');
+
+for (let i = 0; i < object.length; i += 1) {
+  const read = `
+  <section class="works-container">
+  <img class="image-grey" src="${object[i].imgMain}" alt="works-image">
+  <div class="multi">
+      <section class="multi-post">
+          <h2 class="multi-post-title">${object[i].title}</h2>
+          <section class="buttons">
+          ${object[i].techMain.map((language) => `<button class="languages">${language}</button>`).join('')}
+          </section>
```

Comment:
- [ ] Great job so far, the styling is a bit different from the design. Kindly fix that,
- [ ] To stick to the design please make the list items/button 4 i.e (ruby on rails, CSS, HTML, javascript).

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/167943885-c9f35997-9af6-4744-b79c-e8da555fa6f2.png)

### Figma's Design
![image](https://user-images.githubusercontent.com/59416136/167943956-383e9743-5f3a-4830-a6ed-31d8d154b1e3.png)

Reply:
Thank you @Wes-Isaac, I had overlooked that.

![image](https://user-images.githubusercontent.com/59999191/167952318-55280345-74ca-4a3a-8c24-7c2a281abafe.png)